gallery/psget/module/RequireLicenseAcceptance.md

```diff
@@ -0,0 +1,213 @@
+---
+ms.date:  2017-06-09
+schema:  2.0.0
+locale:  en-us
```

Comment:
Please remove this to fix the warning.

Reply:
done

---
In reply to: [136433606](https://github.com/PowerShell/PowerShell-Docs/pull/1620#discussion_r136433606) [](ancestors = 136433606)